packages/obonode/obojobo-sections-assessment/components/rubric/editor-component.js

```diff
@@ -16,19 +17,36 @@ class Rubric extends React.Component {
 	constructor(props) {
 		super(props)
 
+		const content = this.props.element.content
+		this.state = {
+			passingAttemptScore: content.passingAttemptScore || '',
+			passedResult: content.passedResult || '',
+			failedResult: content.failedResult || '',
+			unableToPassResult: content.unableToPassResult || ''
```

Comment:
Since `failedResult` and `unableToPassResult` are 0 by default, it seems to evaluate them as falsy and chooses `''` instead for those values. This can cause issues if those values are not overridden in the editor and you take an assessment that meets failing criteria. 

I think that instead of `''` we should give them the current default values of 100 for `passingAttemptScore` and `passedResult`, and 0 for `failedResult` and `unableToPassResult` to prevent that error from occurring.